### PR TITLE
Added competiton signup help email

### PR DESCRIPTION
**Pull request by Tyler-Ward**
Request volunteer assistance with competition preparation

Fixes https://github.com/srobo/tasks/issues/180

### CODE REVIEW

**Comment by RealOrangeOne** [comment]
We probably want more information than simply "there's a hack day". Timing, location, contacts, stuff like that are really useful to people! Especially for new volunteers, or those not at Southampton.

**Comment by trickeydan** [comment]
I'm happy for this info to be plaguarised from the SRO email I sent on Thursday.

File: SR2019/2019-02-23-hackday-techday-signup.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+to: All Volunteers
+subject: Competition preparation
+preview-text: We need your help to prepare for the competition.
+---
+
+Hi,
+
+The competition is getting nearer and there are many things to do, we have a list of [tasks][task-list] which will need doing before the competition.
+If you have some spare time and can give us a hand your help would be much appreciated and will help us run an even better competition.
```
Comment (Adimote) [inline SR2019/2019-02-23-hackday-techday-signup.md]: add comma
```suggestion
If you have some spare time and can give us a hand, your help would be much appreciated and will help us run an even better competition.
```

File: SR2019/2019-02-23-hackday-techday-signup.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+Come along to the [tech day][techday-info] on the 9th in either London or Southampton where we will also be doing preparation for the competition along with assisting competitors.
+If you are coming along let us know by signing up [here][techday-signup]).
+
+##Take on a task on the task list.
+
+You can also take on some tasks on the task list on github.
+Post a reply on the task so we know you are working on it
+Don't worry if you don't know everything about a task just ask in the task for any help you need.
+
+##Competition signups
```
Comment (trickeydan) [inline SR2019/2019-02-23-hackday-techday-signup.md]:
```suggestion
## Competition signups
```

File: SR2019/2019-02-23-hackday-techday-signup.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+We will be in Mountbatten (building 53) at the university on Level 4 from 10am onwards.
+There will also be a social event in the evening, to be decided during the day.
+A rough agenda for the day can be found [here][hack-day-agenda] and will be updated as more things are planned.
+
+##Tech Day
+
+Come along to the [tech day][techday-info] on the 9th in either London or Southampton where we will also be doing preparation for the competition along with assisting competitors.
+If you are coming along let us know by signing up [here][techday-signup]).
+
+##Take on a task on the task list.
```
Comment (trickeydan) [inline SR2019/2019-02-23-hackday-techday-signup.md]:
```suggestion
## Take on a task on the task list.
```

File: SR2019/2019-02-23-hackday-techday-signup.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+There are several ways you can get involved and help out.
+
+##Southampton preparation day
+
+We will be having a preparation day in Southampton on the 2nd of March where we are planning on doing a load of the tasks.
+We will be in Mountbatten (building 53) at the university on Level 4 from 10am onwards.
+There will also be a social event in the evening, to be decided during the day.
+A rough agenda for the day can be found [here][hack-day-agenda] and will be updated as more things are planned.
+
+##Tech Day
```
Comment (trickeydan) [inline SR2019/2019-02-23-hackday-techday-signup.md]:
```suggestion
## Tech Day
```

File: SR2019/2019-02-23-hackday-techday-signup.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+preview-text: We need your help to prepare for the competition.
+---
+
+Hi,
+
+The competition is getting nearer and there are many things to do, we have a list of [tasks][task-list] which will need doing before the competition.
+If you have some spare time and can give us a hand, your help would be much appreciated and will help us run an even better competition.
+There are several ways you can get involved and help out.
+
+##Southampton preparation day
```
Comment (trickeydan) [inline SR2019/2019-02-23-hackday-techday-signup.md]:
```suggestion
## Southampton preparation day
```

File: SR2019/2019-02-23-hackday-techday-signup.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+
+We will be having a preparation day in Southampton on the 2nd of March where we are planning on doing a load of the tasks.
```
Comment (trickeydan) [inline SR2019/2019-02-23-hackday-techday-signup.md]:
```suggestion
We will be having a preparation day in Southampton on Saturday 2nd of March where we are planning on doing a load of the tasks.
```
I think it's important to mention the day of the week, it helps people know if they are available.

File: SR2019/2019-02-23-hackday-techday-signup.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
+---
+to: All Volunteers
+subject: Competition preparation
+preview-text: We need your help to prepare for the competition.
+---
+
+Hi,
+
+The competition is getting nearer and there are many things to do, we have a list of [tasks][task-list] which will need doing before the competition.
```
Comment (trickeydan) [inline SR2019/2019-02-23-hackday-techday-signup.md]:
```suggestion
The competition is getting nearer and there are many things to do, we have a large list of [tasks][task-list] which will need doing before the competition.
```

File: SR2019/2019-02-23-hackday-techday-signup.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+The competition is getting nearer and there are many things to do, we have a large list of [tasks][task-list] which will need doing before the competition.
+If you have some spare time and can give us a hand, your help would be much appreciated and will help us run an even better competition.
+There are several ways you can get involved and help out.
+
+## Southampton preparation day
+
+We will be having a preparation day in Southampton on Saturday 2nd of March where we are planning on doing a load of the tasks.
+We will be in Mountbatten (building 53) at the university on Level 4 from 10am onwards.
+There will also be a social event in the evening, to be decided during the day.
+A rough agenda for the day can be found [here][hack-day-agenda] and will be updated as more things are planned.
```
Comment (Adimote) [inline SR2019/2019-02-23-hackday-techday-signup.md]: [nit] as a general rule, it's a bad idea to use 'here' as the name of a link, as it's bad for screen readers, and using a more descriptive name makes it easier to skim read the email.
```suggestion
There is a [rough agenda for the day][hack-day-agenda] which will be updated as more things are planned.
```

File: SR2019/2019-02-23-hackday-techday-signup.md [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+
+## Tech Day
+
+Come along to the [tech day][techday-info] on the 9th in either London or Southampton where we will also be doing preparation for the competition along with assisting competitors.
+If you are coming along let us know by signing up [here][techday-signup]).
```
Comment (Adimote) [inline SR2019/2019-02-23-hackday-techday-signup.md]: [nit] as a general rule, it's a bad idea to use 'here' as the name of a link, as it's bad for screen readers, and using a more descriptive name makes it easier to skim read the email.
```suggestion
If you are coming along let us know by [signing up to attend][techday-signup]).
```

File: SR2019/2019-02-23-hackday-techday-signup.md [hunk 10]
```diff
@@ -0,0 +1,40 @@
+---
+to: All Volunteers
+subject: Competition preparation
+preview-text: We need your help to prepare for the competition.
+---
+
+Hi,
+
+The competition is getting nearer and there are many things to do, we have a large list of [tasks][task-list] which will need doing before the competition.
```
Comment (Adimote) [inline SR2019/2019-02-23-hackday-techday-signup.md]: [nit] this link can be more descriptive

```suggestion
The competition is getting nearer and there are many things to do, we have a large [list of tasks][task-list] which will need doing before the competition.
```

File: SR2019/2019-02-23-hackday-techday-signup.md [hunk 11]
```diff
@@ -0,0 +1,40 @@
...
+
+## Tech Day
+
+Come along to the [tech day][techday-info] on the 9th in either London or Southampton where we will also be doing preparation for the competition along with assisting competitors.
+If you are coming along let us know by signing up [here][techday-signup]).
+
+## Take on a task on the task list.
+
+You can also take on some tasks on the task list on github.
+Post a reply on the task so we know you are working on it
```
Comment (Adimote) [inline SR2019/2019-02-23-hackday-techday-signup.md]: [nit] comma
```suggestion
Post a reply on the task so we know you are working on it,
```

**Comment by Adimote** [comment]
Email scheduled for 2019-02-24 10:00 GMT:
https://mailchi.mp/eb78175db43b/competition-preparation

**Pull request merged**